### PR TITLE
storage: refine some code and comments of the storage pkg

### PR DESCRIPTION
**Pull request by JmPotato**
Signed-off-by: JmPotato <ghzpotato@gmail.com>

### What problem does this PR solve?

Part of #3453. Refine some code and comments of the `storage` pkg.

### What is changed and how it works?

* Add more comments.
* Refine some code.

### Check List

Tests

- Unit test
- Integration test

### Release note

```release-note
None.
```

### CODE REVIEW

File: server/storage/leveldb_backend.go [hunk 1]
```diff
@@ -127,46 +131,46 @@ func (lb *levelDBBackend) SaveRegion(region *metapb.Region) error {
...
-	return nil
-}
-
 func (lb *levelDBBackend) DeleteRegion(region *metapb.Region) error {
 	return lb.Remove(endpoint.RegionPath(region.GetId()))
 }
 
-// FlushRegion saves the cache region to region storage.
-func (lb *levelDBBackend) FlushRegion() error {
+// Flush saves the cache region to the underlying storage.
```
Comment (rleungx) [inline server/storage/leveldb_backend.go:138]: Why change the name?

Comment (JmPotato) [reply server/storage/leveldb_backend.go:138]: It should re-implement the `Flush` method of the `endpoint.Region Storage`. I forgot to rename it before.

**Review by rleungx — APPROVED** [review]

**Review by HunDunDM — APPROVED** [review]

**Pull request merged**